src/bin/ion/commands/beta/generate/utils.rs

```diff
@@ -29,6 +29,17 @@ pub struct Field {
     pub(crate) isl_type_name: String,
 }
 
+/// Represents an anonymous type that can be a part of another type definition.
+/// This will be used by the template engine to add these intermediate data models for anonymous types
+/// in to the parent type definition's module/namespace.
+#[derive(Serialize)]
+pub struct AnonymousType {
```

Comment:
Suggestion—the more I've thought about it, the more I think this change isn't really about _anonymous_ types. It's about _nested_ types (which, for now, happen to all be anonymous). I think this PR would be a lot more readable if you changed this struct, and the terminology throughout, to talk about "nested" things.

Maybe the ISL terminology needs to be changed too.

Reply:
That's a good suggestion. I will change that.

Reply:
Changed all references of `anonymous type` to `nested type`.